sentry-spring-boot-jakarta/src/main/java/io/sentry/spring/boot/jakarta/SentryAutoConfiguration.java

```diff
@@ -300,7 +294,7 @@ public FilterRegistrationBean<SentryTracingFilter> sentryTracingFilter(
       @Configuration(proxyBeanMethods = false)
       @ConditionalOnClass(HandlerExceptionResolver.class)
       @Open
-      static class SentryExceptionResolverConfigurationWrapper {
+      static class SentryNonServletOnlyModeConfig {
```

Comment:
Just a suggestion, but wouldn't something like `SentryMvcModeConfig`, be more easily understandable?

Reply:
yeah probably